examples/cd_model_unc_cifar10.ipynb

```diff
@@ -0,0 +1,752 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Model-uncertainty based drift detection on CIFAR-10\n",
```

Comment:
Change title to make clear both classifier uncertainty drift on CIFAR10 and regression uncertainty drift on Wine datasets are illustrated.